README.md

```diff
@@ -1,5 +1,121 @@
-# Balance Tetris
+<div style="background-color: #FFFFFF">
+    <p align="center">
+    <a href="https://github.com/WoochanLee/BalanceTetris">
+        <img src="https://github.com/WoochanLee/BalanceTetris/blob/master/res/young.png?raw=true" height="300" />
+        <img src="https://github.com/WoochanLee/BalanceTetris/blob/master/res/chan.png?raw=true" height="300" />
+    </a>
+    </p>
+    <br>
+    <p align="center">
+        <img src="https://github.com/WoochanLee/BalanceTetris/blob/master/res/logo.png?raw=true" width="600"/>
+    </p>
+    <p align="center" style="color: #000; font-size: 25px">🕹️You Can Single Play Online👇</p>
+    <div align="center" style="margin-bottom: 40px">
+        <a href="https://github.com/WoochanLee/BalanceTetris" style="color: blue; font-size: 25px">
+            <u>https://github.com/WoochanLee/BalanceTetris</u>
+        </a>
+    </div>
+    <p align="center">
+        <img src="https://img.shields.io/badge/license-GPL%20v3-black.svg">
+        <img src="https://img.shields.io/badge/release-alpha-black.svg">
+    <br />
+</div>
+<br>
+<br>
 
-Made by Woody
+# Features 🎮
 
-Play Tetris Link : [https://woochanlee.github.io/BalanceTetris/](https://woochanlee.github.io/BalanceTetris/)
+Easy to Play: You can play Tetris anywhere with a computer web browser.
+<br>
+Using T-spin: You can use T-spin technic.
+<br>
+Cats are cute: Cats are cute.
+<br>
+<br>
+<br>
+
+# How to Play 💁
```

Comment:
In future, This part should be changed as 'image' indicates how to play. But not required. Up to you!